turbo/stream_templ.go

```diff
@@ -8,10 +8,12 @@ import "github.com/a-h/templ"
 import "context"
 import "io"
 
-func actionTemplate(action string, target string, template templ.Component) templ.Component {
+func actionTemplate(action string, target string) templ.Component {
 	return templ.ComponentFunc(func(ctx context.Context, w io.Writer) (err error) {
 		ctx, _ = templ.RenderedCSSClassesFromContext(ctx)
 		ctx, _ = templ.RenderedScriptsFromContext(ctx)
+		var_1 := ctx
```

Comment:
This was to prevent children being used in components that are nested, I separated it into its own context so that only the { children... } further down can access that context.

If that makes sense.

Otherwise if another templElement was rendered in this component it would get the children passed in, which is bad.